src/Mod/Material/App/MaterialLoader.cpp

```diff
@@ -29,6 +29,10 @@
 #include <QList>
 #include <QMetaType>
 
+#if QT_VERSION >= QT_VERSION_CHECK(6,0,0)
+#include <QtCore5Compat/QRegExp>
```

Comment:
Can the code be modified to use `QRegularExpression` instead? `QRegExp` is very old.

Reply:
Yes. This should be a direct substitution in this code.

Reply:
Are you doing that, or should @leviathanch modify this PR to take that course of action instead?

Reply:
I can do it but that will come in a future PR. This should be fine in the interim.